### ISSUE TITLE
Trojan.YakbeexMSIL.ZZ4  in .exe ?

### ISSUE THREAD

**Issue by raphaelquast** [Twometer/NoFences#12]
according to a virustotal scan of the provided .exe there's the Trojan.YakbeexMSIL.ZZ4 coming along!
https://www.virustotal.com/gui/file/bb166429ea8d901ab60795b79e58ac1615f206291081095d4814c388683c8e37/detection

**Comment by Twometer** [comment]
Same issue as #10 - Some antiviruses falsely detect unsigned/"homemade" software as a virus. The .exe is a straight build from the source :)

**Comment by raphaelquast** [comment]
well... but there's a lot of unsigned software out there that does not get clear warning for a specific troyan... unfortunately for me that's enough to keep my hands off that .exe for now...

**Comment by Twometer** [comment]
As someone who cares about security as well, I definitely understand your concerns. But I mean, the warning about that Trojan is from one no-name antivirus, none of the big ones detected it - and I have seen *a lot* of open-source/unsigned software that was completely clean and had much worse detection rates at VirusTotal, even with the big antiviruses. 

But, if you don't trust the .exe file and would still like to try the app, you can build the source code from the repository yourself with Visual Studio. Or you can use a tool like [dnspy](https://github.com/dnSpy/dnSpy) to decompile the .exe and check for yourself. Also, the VirusTotal behavior analysis gives you some insight in what the app does.

I suspect that the false-positive detection comes from the many WinApi calls that are made by NoFences in order to provide full functionality.

**Comment by raphaelquast** [comment]
I agree that only little-known antivirus tools seem to complain about your file...
unfortunately my C knowledge is quite limited and I certainly don't have the time to dig into this deeper... I was just curious since my desktop tends to become a mess from time to time 😄 

... another online sandbox-scan tool at least confirms that there is no immediate suspicious network activity 
https://www.hybrid-analysis.com/sample/bb166429ea8d901ab60795b79e58ac1615f206291081095d4814c388683c8e37/611e4a7442136e68e2583781

it still terms it malicious but only because of third-party scans (e.g. virustotal)
also the "risk-assessment" they provide seems to reveal only expected behaviour...

- Fingerprint
    - Queries kernel debugger information
    - Reads the active computer name
    - Reads the cryptographic machine GUID
    - Reads the windows installation language 
- Evasive
    - Possibly tries to implement anti-virtualization techniques
    - Tries to sleep for a long time (more than two minutes)  

It's a pity that this whole analysis is so inconclusive... but I agree that this looks more like a false-positive than an actual threat...
(especially since the 2 companies look quite suspicious themselves 😅 )
